binding/unity/Assets/Porcupine/Porcupine.cs

```diff
@@ -408,18 +408,22 @@ private static string GetDefaultModelPath()
         private static Dictionary<BuiltInKeyword, string> GetBuiltInKeywordPaths(string platform)
         {
 #if !UNITY_EDITOR && UNITY_ANDROID
-            string keywordFilesDir = Path.Combine(Path.Combine(Application.persistentDataPath, "keyword_files"), platform);
+            string keywordFilesDir = Path.Combine(Application.persistentDataPath, "keyword_files", platform);
```

Comment:
I know those multi-`Path.Combine`s look weird, but they actually make it compatible with Unity 2017.1 because they conform to an earlier version of C#. Let's keep them there.